### PR TITLE
Patch release process description

### PR DESCRIPTION
**Pull request by atalman**
Patch release process description

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -41,6 +42,7 @@ Following is the Release Compatibility Matrix for PyTorch releases:
 
 | PyTorch version | Python | Stable CUDA | Experimental CUDA |
 | --- | --- | --- | --- |
+| 2.0.1 | >=3.8, <=3.11 | CUDA 11.7, CUDNN 8.5.0.96 | CUDA 11.8, CUDNN 8.7.0.84 |
```
[the remark this answers is not in the document]
Comment (atalman) [reply RELEASE.md:45]: True let me remove this!

File: RELEASE.md [hunk 2]
```diff
@@ -237,6 +239,21 @@ Patch releases should be considered if a regression meets the following criteria
 
 ## Patch Release Process
 
+### Patch Rlease Process Description
```
Comment (seemethere) [inline RELEASE.md]:
```suggestion
### Patch Release Process Description
```

File: RELEASE.md [hunk 3]
```diff
@@ -237,6 +239,21 @@ Patch releases should be considered if a regression meets the following criteria
...
+
+> Main POC: Patch Release Managers, Triage Reviewers
+
+Patch releases should follow these high-level phases. This process starts immediately after the previous release has completed.
+Minor release process takes around 6-7 weeks to complete.
+
+1. Triage, is a process where issues are identified, graded, compared to Patch Release Criteria and added to Patch Release milestone. This process normally takes 2-3 weeks after the release completion.
+2. Patch Release: Go/No Go meeting between PyTorch Releng, PyTorch Core and Project Managers where potential issues triggering a release in milestones are reviewed and following decisions are made:
+ - Should the new patch Release be created
```
Comment (seemethere) [inline RELEASE.md]:
```suggestion
 - Should the new patch Release be created?
```

File: RELEASE.md [hunk 4]
```diff
@@ -237,6 +239,21 @@ Patch releases should be considered if a regression meets the following criteria
...
+ - Timeline execution for the patch Release
```
Comment (seemethere) [inline RELEASE.md]:
```suggestion
 - Timeline execution for the patch release
```

File: RELEASE.md [hunk 5]
```diff
@@ -237,6 +239,21 @@ Patch releases should be considered if a regression meets the following criteria
...
+3. Cherry picking phase. Starts after the decision is made to create patch Release. At this point a new Issue tracker for patch release is created, and announcement is made on official channels [example announcement](https://dev-discuss.pytorch.org/t/pytorch-release-2-0-1-important-information/1176). The authors of the regressions are asked to create cherry picks. This process normally takes 2 weeks.
```
Comment (seemethere) [inline RELEASE.md:253]:
```suggestion
3. Cherry picking phase starts after the decision is made to create patch release. At this point a new release tracker for the patch release is created, and an announcement will be made on official channels [example announcement](https://dev-discuss.pytorch.org/t/pytorch-release-2-0-1-important-information/1176). The authors of the fixes to regressions will be asked to create their own cherry picks. This process normally takes 2 weeks.
```

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**